legacy/src/api/assets/assets.md

```diff
@@ -114,14 +114,15 @@ Tokens have the following fields along with the base asset fields.
 
 {% h4 Fields %}
 
-|   Field   |             Type             |                                         Description                                          |
-| :-------- | :--------------------------- | :------------------------------------------------------------------------------------------- |
-| value     | Array {% opt %}              | The [Monetary Amounts][] representing the token's nominal value in its supported currencies. |
-| validFrom | {% dt Timestamp %} {% opt %} | The date when the asset becomes spendable.                                                   |
-| expiresAt | {% dt Timestamp %} {% opt %} | The date when the asset expires.                                                             |
-| img       | String {% opt %}             | The img URL of the token.                                                                    |
-| brandName | String {% opt %}             | The name of the brand that the token belongs to.                                             |
-| brandImg  | String {% opt %}             | The img URL of the brand that the token belongs to.                                          |
+|     Field     |             Type             |                                         Description                                          |
+| :------------ | :--------------------------- | :------------------------------------------------------------------------------------------- |
+| value         | Array {% opt %}              | The [Monetary Amounts][] representing the token's nominal value in its supported currencies. |
+| validFrom     | {% dt Timestamp %} {% opt %} | The date when the asset becomes spendable.                                                   |
+| expiresAt     | {% dt Timestamp %} {% opt %} | The date when the asset expires.                                                             |
+| img           | String {% opt %}             | The img URL of the token.                                                                    |
+| issuer        | String {% opt %}             | The identifier for the issuer of the token.                                                  |
+| issuerWebsite | String {% opt %}             | The URL of the issuer of the token.                                                          |
+| issuerImg     | String {% opt %}             | The img URL of the issuer that the token belongs to.                                         |
```

Comment:
Does this need description?

Reply:
Description is above in the `Asset` section.